nixos/modules/system/boot/networkd.nix

```diff
@@ -2835,6 +2843,7 @@ let
         "systemd-networkd-wait-online.service"
         "systemd-networkd.service"
         "systemd-networkd.socket"
+        "systemd-networkd-persistent-storage.service"
```

Comment:
[networkctl](https://www.freedesktop.org/software/systemd/man/latest/networkctl.html#%0A%20%20%20%20%20%20%20%20%20%20persistent-storage%0A%20%20%20%20%20%20%20%20%20%20BOOL%0A%20%20%20%20%20%20%20%20) has the slim documents. Source is https://raw.githubusercontent.com/systemd/systemd/main/units/systemd-networkd-persistent-storage.service.